### PR TITLE
Python 3 support.

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -43,11 +43,11 @@
     ],
     extras_require=dict(
         test=[
-            'zope.app.publisher',
+            'zope.browserpage',
             'zope.app.wsgi',
             'zope.configuration',
             'zope.principalregistry',
-            'zope.testbrowser',
+            'zope.testbrowser>=5.0.0.dev0',
```
Comment (mgedmin) [inline setup.py]: 5.0.0 final is on pypi, so I suggest dropping the .dev0 here

File: src/zope/globalrequest/README.rst [hunk 2]
```diff
@@ -131,6 +129,5 @@ If we now provide a request value we should be greeted properly:
 
 Once the request has been processed, it should not be available anymore:
 
-  >>> print getRequest()
-  None
-
+  >>> print(getRequest())
+  None
```
Comment (mgedmin) [inline src/zope/globalrequest/README.rst]: Please don't remove the trailing newline.

**Review by mgedmin — COMMENTED** [review]
LGTM.

I haven't looked yet at the Travis failures.

**Comment by mgedmin** [comment]
Huh, travis failed to find zope.testbtowser >= 5.0.0.dev0 on PyPI.  Strange.  Retry the build?

**Comment by dhavlik** [comment]
Dropped the .dev0 in setup.py, re-added the trailing newline in the doctest and added a changelog entry.

**Review by mgedmin — APPROVED** [review]

**Pull request merged**